### DISCUSSION TITLE
part-ii/2011-100/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#159] [Question comments]
Part II 2011 Number Theory: Paper 4, Section $I$, I 

https://questions.tripos.org/part-ii/2011-100/

**Comment by walrus1729** [comment]
<details>
<summary>Solution</summary>

(i) Suppose there are finitely many primes $p_1, p_2, \dots, p_k$. Then set $N = p_1 p_2 \dots p_k + 1$, which is divisible by none of the primes on our list, but since every integer greater than $1$ is divisible by at least one prime, there must be a prime not our list, so we have a contradiction.

(ii) The sequence $N! + 2, N! + 3, \dots, N! + N$ is a sequence of $N-1$ consecutive composite integers (for $N > 1$), since $k$ divides $N! + k$ for $2 \leq k \leq N$. Thus we can have a gap of at least $N$ between consecutive primes, and we can take $N$ arbitrarily large.

</details>
